packages/@okta/i18n/src/properties/login.properties

```diff
@@ -971,10 +971,14 @@ oie.email.challenge.mfa.title = Get a verification email
 oie.email.verify.primaryButton = Send me an email
 # {0} is a user's email address
 oie.email.verify.subtitle.text.with.email = Send a verification email to <$1>{0}</$1> by clicking on "Send me an email".
+# {1} is a user's secondary email address
+oie.email.verify.subtitle.text.with.email.and.secondary.email = Send a verification email to <$1>{0}</$1> and <$2>{1}</$2> by clicking on "Send me an email".
 oie.email.verify.subtitle.text.without.email = Send a verification email by clicking on "Send me an email".
 # {0} is a user's email address
-oie.email.verify.alternate.magicLinkToEmailAddress = We sent an email to <$1>{0}</$1>. 
-oie.email.verify.alternate.magicLinkToYourEmail = We sent you a verification email. 
+oie.email.verify.alternate.magicLinkToEmailAddress = We sent an email to <$1>{0}</$1>.
+# {1} is a user's secondary email address
+oie.email.verify.alternate.magicLinkToEmailAddress.with.secondary.email = We sent an email to <$1>{0}</$1> and <$2>{1}</$2>.
```

Comment:
nit: Could you add comment for {0} variable as well

Reply:
Done.